frontend/views/components/CalloutCard.vue

```diff
@@ -2,7 +2,7 @@
   section.c-calloutCard(:class='{ card: isCard }')
     component.c-svg(:is='svg')
     div
-      i18n.is-title-3.c-title(tag='h3') {{ title }}
+      i18n.is-title-3.c-title(tag='h3' v-if='title') {{ title }}
```

Comment:
This shouldn't be an `i18n` tag since we're passing in a variable here via `{{ title }}` that should already be translated